### ISSUE TITLE
Extent Report in HP Quality Center

### ISSUE THREAD

**Issue by petuk** [anshooarora/extentreports#440]
Hi,

could you help me how I can integrate the extent report result in the HP Quality Center? I mean if the TC is passed in the extent Report, just automatically set in the HP QC? 

Thanks for you answer!

**Comment by anshooarora** [comment]
This is a QC question, and out of scope for Extent.  This would require building another module that communicates with OTA and performs these updates.

**Comment by guneetgarg** [comment]
@anshooarora that would be great if we have any such module that can change QC pass/fail status 
and attach the failure message with Test Case

**Comment by petuk** [comment]
Yes, I think so :) As I read on the internet, there are some OTA communication things

**Comment by anshooarora** [comment]
I am open to this, and can mark this as an enhancement.  What I feel would be ideal is creating a plugin that integrates with Extent, similar to what Vimal has created for Cucumber:  https://github.com/email2vimalraj/CucumberExtentReporter

**Comment by petuk** [comment]
Thanks in advance.
